### PR TITLE
805 20220203 print pdf stack

### PR DESCRIPTION
**Pull request by stolman-digitpaint**
Add timeline PDF printer based on javascript

### CODE REVIEW

File: lib/GADS.pm [hunk 1]
```diff
@@ -1660,7 +1659,7 @@ post '/print' => require_login sub {
     );
 };
 
-prefix '/:layout_name' => sub {
+prefix '/:layout_name' => sub {data_calendar
```
Comment (abeverley) [inline lib/GADS.pm:1663]: Thanks for the PR @stolman-digitpaint - I'm just checking whether the above ```data_calendar``` is meant to be there? It doesn't look quite right.

Comment (stolman-digitpaint) [reply lib/GADS.pm:1663]: it's not supposed to be there, no. I've removed this issue earlier, but there was a merge conflict when i pushed some finalizing local changes, and in the resulting merge they seem to have come back. i'll fix it shortly

Comment (stolman-digitpaint) [reply lib/GADS.pm]: issue is resolved

**Comment by abeverley** [comment]
Thanks @stolman-digitpaint looking great. We'll do some final thorough testing but looks good on the face of it.

**Pull request merged**